blueskyacademy/bluesky-nextjs
# rework contact page and add class page

## Description
- Rework Contact Page
- Rework Register School Tour
- Add Class Page


## Why
- User can visit Class Page (Kindergarten, Primary, Secondary)
- Users can register for a school tour


## How to QA
- Users can register for a school tour
- Users can visit the class page to view detailed program, why parents should choose BSA, stats, and teachers



## Screenshots
![Screen Shot 2021-12-24 at 15 09 15](https://user-images.githubusercontent.com/74655110/147332971-a2399ee5-a723-4f3f-a831-f33ad99aecde.png)
![Screen Shot 2021-12-24 at 15 09 35](https://user-images.githubusercontent.com/74655110/147332977-f33958cd-3621-42cc-a50d-dc9ce92e01b2.png)
